src/Constants.ts

```diff
@@ -46,6 +46,7 @@ export enum ErrorCodes {
     USER_DEVICE_KEY_DECRYPTION_FAILURE = 208,
     USER_PASSCODE_CHANGE_FAILURE = 209,
     USER_DEVICE_DELETE_REQUEST_FAILURE = 210,
+    USER_PRIVATE_KEY_ROTATION_FAILURE = 211,
```

Comment:
This also needs to be added to the `ErrorCodes` enum in the `ironweb.d.ts` file.